pkg/iac-providers/kustomize/v3/load-dir.go

```diff
@@ -24,28 +24,29 @@ func (k *KustomizeV3) LoadIacDir(absRootDir string) (output.AllResourceConfigs,
 
 	files, err := utils.FindFilesBySuffixInDir(absRootDir, KustomizeFileNames())
 	if err != nil {
+		err = errorKustomizeNotFound(err)
 		zap.S().Error("error while searching for iac files", zap.String("root dir", absRootDir), zap.Error(err))
 		return allResourcesConfig, err
 	}
 
 	if len(files) == 0 {
-		err = errors.New("could not find a kustomization.yaml/yml file in the directory")
+		err = errorKustomizeNotFound(errors.New("could not find a kustomization.yaml/yml file in the directory"))
```

Comment:
Recommend to move the error definitions into a variable that can be referenced easily by unit tests